translations/en/LC_MESSAGES/messages.po

```diff
@@ -19,133 +19,133 @@ msgstr ""
+msgstr "`{invalid_argument}` and `{invalid_argument_2}` cannot be used with `{command}` because one is {invalid_type} and the other is {invalid_type_2}. Try changing `{invalid_argument}` to {invalid_type_2} or `{invalid_argument_2}` to {invalid_type}."
 
 msgid "Lonely Echo"
-msgstr "You used an `{echo}` before an `{ask}`, or an `{echo}` without an `{ask}`. Place an `{ask}` before the `{echo}`."
+msgstr "Command `{echo}` used before or without using an `{ask}`. Try writing `{ask}` before the `{echo}`."
 
 msgid "Lonely Text"
-msgstr "It looks like you forgot to use a command with the text you used in line {line_number}"
+msgstr "It looks like I am missing a command with the text that was used in line {line_number}. Try writing the appropiate command with the text."
 
 msgid "Missing Additional Command"
-msgstr "It looks like you forgot to complete writing `{command}` on line {line_number}. Try adding `{missing_command}` to your code."
+msgstr "Complete writing the `{command}` on line {line_number}. Try adding `{missing_command}` to your code."
 
 msgid "Missing Colon Error"
-msgstr "Starting at level 17, `{command}` needs a `:`. It looks like you forgot to use one at the end of line {line_number}."
+msgstr "Starting at level 17, `{command}` needs a `:`. Try using one at the end of line {line_number}."
 
 msgid "Missing Command"
-msgstr "It looks like you forgot to use a command on line {line_number}."
+msgstr "I need a command on line {line_number}. Try looking for the exercise section to find which command to use."
 
 msgid "Missing Inner Command"
-msgstr "It looks like you forgot to use a command with the `{command}` statement you used on line {line_number}."
+msgstr "I need a command for the `{command}` statement on line {line_number}. Try looking for the exercise section to find which command to use."
 
 msgid "Missing Square Brackets"
-msgstr "It looks like you forgot to use square brackets `[]` around the list you were creating on line {line_number}."
+msgstr "List you created on line {line_number} is missing square brackets `[]`. Try using square brackets `[]` for lists."
 
 msgid "Missing Variable"
-msgstr "It looks like your `{command}` is missing a variable at the start of the line."
+msgstr "Command `{command}` is missing a variable at the start of the line. Try writing the variable before the `{command}`"
 
 msgid "Misspelled At Command"
-msgstr "It looks like you might have misspelled the `{command}` command, instead you wrote `{invalid_argument}` in line {line_number}."
+msgstr "`{invalid_argument}` seems to have a spelling mistake on line {line_number}. Try writing `{command}` instead."
 
 msgid "No Indentation"
-msgstr "You used too few spaces in line {line_number}. You used {leading_spaces} spaces, which is not enough. Start every new block with {indent_size} spaces more than the line before."
+msgstr "Insufficent {leading_spaces} spaces used on line {line_number}. Try increasing by {indent_size} for each new block."
```

Comment:
Insufficient is a complex word! Maybe we should include target audience (10 and up) explicitly in the guidelines too and live by that?